stories/Tab.stories.ts

```diff
@@ -0,0 +1,71 @@
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { TabComponent } from 'projects/ion/src/lib/tab/tab.component';
```

Comment:
```suggestion
import { TabComponent } from '../projects/ion/src/lib/tab/tab.component';
```
This import is breaking the build when you run storybook.